oap-server/server-alarm-plugin/src/test/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReaderTest.java

```diff
@@ -18,16 +18,23 @@
 
 package org.apache.skywalking.oap.server.core.alarm.provider;
 
-import java.util.List;
+import org.apache.skywalking.oap.server.core.alarm.provider.dingtalk.DingtalkSettings;
 import org.junit.Assert;
 import org.junit.Test;
 
-public class AlarmRuleInitTest {
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.any;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+
+public class RulesReaderTest {
     @Test
-    public void testInit() {
+    public void testReadRules() {
```

Comment:
But I have noticed this, maybe above is just a coverage issue? FYI @kezhenxu94, any clue?